README.rst

```diff
@@ -0,0 +1,55 @@
+====================
+dspace-python-client
+====================
+
+A Python client library for working with the DSpace API. This was written against the `DSpace version 6.x API <https://wiki.lyrasis.org/display/DSDOC6x/REST+API>`_ and will not work with later versions of DSpace.
+
+-----
+Usage
+-----
+
+This project has not been released for active use yet. Check back soon!
+
+------------
+Development
+------------
+
+^^^^^^^^^^^^
+Dependencies
+^^^^^^^^^^^^
+~~~~~~
+Python
+~~~~~~
+
+Version 3.9
+
+~~~~~~
+Poetry
+~~~~~~
+This project uses Poetry for dependency management and packaging.
+
+You only need to install Poetry once, so if you've already installed it for another project you can skip this step. Poetry will automatically pick up the current Python version and use it to create virtualenvs accordingly.
+
+To install poetry:
+
+* Mac/unix::
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+* Windows::
+
+    (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
+
+Run ``poetry --version`` to confirm installation.
+
+See `Poetry installation docs <https://python-poetry.org/docs/#installation>`_ for configuration options and more information.
+
+^^^^^^^^^^^^
+Installation
+^^^^^^^^^^^^
+
+Install dspace-python-client for local development by running::
+
+  git clone https://github.com/MITLibraries/dspace-python-client.git
+  cd dspace-python-client
+  make install
```

Comment:
We chatted on slack but at some point we may want to document how to use the correct python version (or at least gently remind that if you don't have python versioning working locally that any issues are out of scope of this project to solve).

Reply:
Yeah I'm not opposed to this although 1) I'm hoping the addition of the .python-version file in a future PR will help, and 2) I'd rather do it in a future PR.